ios/Classes/Util/ResultConverter.m

```diff
@@ -1,8 +1,8 @@
-#import "ScanResultFactory.h"
+#import "ResultConverter.h"
 
-@implementation ScanResultFactory
+@implementation ResultConverter
```

Comment:
It's not aware of any result, it just gets a serializable object. `JSONStringifier`?

Reply:
Changed to JSONStringifier